### PR TITLE
macos 10.13 software updates week 17

### PR DESCRIPTION
**Pull request by al-cheb**
Software Updates Week 17:
- Homebrew 2.1.1
- Gradle 5.4
- java 1.7 : OpenJDK Runtime Environment (Zulu 7.29.0.5-CA-macosx) (build 1.7.0_222-b08)
- java 1.8 : OpenJDK Runtime Environment (Zulu 8.38.0.13-CA-macosx) (build 1.8.0_212-b04) (default)
- java 11 : OpenJDK Runtime Environment Zulu11.31+11-CA (build 11.0.3+7-LTS)
- java 12 : OpenJDK Runtime Environment Zulu12.2+3-CA (build 12.0.1+12)
- Apache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T19:00:29Z)
- Go 1.12.4
- macOS toolcache Python 2.7.16 3.4.8 3.5.7 3.6.8 3.7.3
- macOS toolcache Ruby 2.3.8 2.4.6 2.5.5 2.6.2

### CODE REVIEW

File: images/macos/macos-10.13-Readme.md [hunk 1]
```diff
@@ -2,12 +2,13 @@
 
 The following software is installed on machines in the Azure Pipelines **macOS-10.13** VM image ('Hosted macOS High Sierra' pool).
 
-## Upcoming Image Changes
+## Majority Image Changes
```
Comment (ncouraud) [inline images/macos/macos-10.13-Readme.md]: Why the wording change here?

File: images/macos/macos-10.13-Readme.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ The following software is installed on machines in the Azure Pipelines **macOS-1
 - Azure-CLI 2.0.62
 
 ### Pre-cached tools
-- Python 2.7.15 3.4.8 3.5.5 3.6.8 3.7.2
+- Python 2.7.16 3.4.8 3.5.7 3.6.8 3.7.3
```
Comment (ncouraud) [inline images/macos/macos-10.13-Readme.md]: Isn't Ruby in the toolcache now as well?

Comment (al-cheb) [reply images/macos/macos-10.13-Readme.md]: We will update getsysteminfo.sh script to add Ruby toolcache info(Week 18) as we do for Python.

File: images/macos/macos-10.13-Readme.md (hunk 1 above)
Comment (al-cheb) [reply images/macos/macos-10.13-Readme.md]: Revert ## Upcoming Image Changes

File: images/macos/macos-10.13-Readme.md [hunk 3]
```diff
@@ -4,10 +4,13 @@ The following software is installed on machines in the Azure Pipelines **macOS-1
 
 ## Upcoming Image Changes
 
+## Majority Image Changes
```
Comment (ncouraud) [inline images/macos/macos-10.13-Readme.md]: Can you drop the additional header?

**Pull request merged**